src/openrct2/rct1/S4Importer.cpp

```diff
@@ -2733,7 +2734,6 @@ namespace RCT1
         dst->num_seats = src->num_seats;
         dst->speed = src->speed;
         dst->powered_acceleration = src->powered_acceleration;
-        dst->brake_speed = src->brake_speed;
```

Comment:
S4 and S6 import call UYppdateTrackBrakeSpeed() which calls GetSpeedFromTrackElement() which always sets brake_speed